### PR TITLE
add music fluff items

### PR DESCRIPTION
**Pull request by KorGgenT**
#### Summary

```SUMMARY: Content "add music fluff items"```
#### Purpose of change

wist was making a band space nested map, and i took it upon myself to add a few items that you would find in such a place. additional places you would find these items: A/V warehouse/closet, recording studio
#### Describe the solution

Adds some music fluff items like xlr cables, mic stands, and a mixer. None of these items have disassembly recipes right now, but I encourage someone else to do that if they wish for these items to do anything. (specifically the mixer can have some electroniocs in it)
#### Describe alternatives you've considered

going acoustic
#### Testing

made sure the game loaded without debug messages

### CODE REVIEW

**Comment by curstwist** [comment]
Looks good to me, I'm not experienced with item creation much though.  It'd be nice if we could use the headphones, or have them wearable with a slight noise reduction.  Maybe a future PR.

**Comment by Mecares** [comment]
Wait a second, we can use the greek alphabet as symbols? Omega is amazing for head phones great idea! If that is the case maybe a big gamma Γ could be better for the microphone stand.

**Comment by KorGgenT** [comment]
oh i didn't think of that, great idea!

File: data/json/items/generic/music.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "color": "dark_gray",
+    "volume": "550 ml",
+    "weight": "265 g"
+  },
+  {
+    "type": "GENERIC",
+    "id": "mic_stand_tall",
+    "name": { "str": "microphone stand" },
+    "description": "A lightweight telescoping boom stand made of aluminum and painted black.  About 6' tall when full extended, or 3' when closed fully.  Has threads for a mic clip on one end.",
+    "symbol": "|",
```
Comment (KorGgenT) [inline data/json/items/generic/music.json:53]:
```suggestion
    "symbol": "Γ",
```

File: data/json/items/generic/music.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    "symbol": "y",
+    "color": "dark_gray",
+    "volume": "800 ml",
+    "weight": "500 g"
+  },
+  {
+    "type": "GENERIC",
+    "id": "mixer_music",
+    "name": { "str": "mixer" },
+    "description": "A device with faders, switches, and knobs that mixes input signal and sends it to two output XLR cables. (left and right)",
```
Comment (kevingranade) [inline data/json/items/generic/music.json:74]:
```suggestion
    "description": "A device with faders, switches, and knobs that mixes input signal and sends it to two output XLR cables.  (left and right)",
```

**Pull request merged**